Networking/Networking/Model/Product/ProductReview.swift

```diff
@@ -0,0 +1,105 @@
+import Foundation
+
+/// Represents a ProductReview Entity.
+///
+public struct ProductReview: Decodable {
+    public let siteID: Int
+    public let reviewID: Int
+    public let productID: Int
+
+    public let dateCreated: Date        // gmt
+
+    public let statusKey: String
```

Comment:
do we still need this property if it's possible to set `status: ProductReviewStatus` in `init(from decoder: Decoder)`?

Reply:
That was my first instinct as well, but I decided to go with the raw String fro two reasons, (sorted by importance, descending 😛 ):
- We'd need to persist this entity in coredata, and I am not sure we really need to persist the ProductReviewStatus as well.
- We are following the same pattern in other objects like Product / ProductType

So I went with persisting the raw status and adding a computed property that returns the ProductReviewStatus. Does it make sense?

Reply:
yea makes sense! I didn't think of how it'd be represented in Core Data just by looking at the networking interface, and following the same pattern sounds great 👍